devtools/maven/src/main/java/io/quarkus/maven/RemoteDevMojo.java

```diff
@@ -48,6 +48,7 @@
 /**
  * The dev mojo, that connects to a remote host
  * <p>
+ * You can launch forked app directly with {@code dev}
```

Comment:
I don't think I would add this sentence. I would just add a final dot to the former one and remove the `<p>`